docs/running/handling_spam.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Handling spam
+---
+
+# Handling spam
+
+<p class="lead">
+  As your Alaveteli site becomes more popular, you may find that spammers begin to target it in various ways. This guide will tell you how Alaveteli supports you in dealing with spam.
+</p>
+
+There are several ways in which spammers can cause problems on an Alaveteli site:
+
+* [Spam emails sent to request addresses](#spam-emails-sent-to-request-addresses)
+* [Spammy user profiles](#spammy-user-profiles)
+* [Spammy annotations](#spammy-annotations)
+
+We will discuss each of these in turn.
+
+## Spam emails sent to request addresses
+
+Spammers can send an email to the special email address for a request, causing that email to appear on the request thread, and a notification to be sent to the requester that there’s a new response to their request.
+
+Spammers can obtain the special email for a request in two ways:
+
+1. Alaveteli's automatic redaction tries to hide all email addresses in the public view of a response made to a request. In some cases  e.g. in some PDF files attached to responses, this may not work. Spammers then could obtain the address from the response displayed on the site.
+
+1. The email address to fall into the hands of spammers from the authority end e.g. the email account of someone at the authority is hacked and spammers get access to the address book.
```

Comment:
This doesn't quite read right, how about...

```diff
- 1. The email address to fall into the hands of spammers [...]
+ 1. The email address falls into the hands of spammers [...]
```

Or...
```diff
- 1. The email address to fall
+ 1. Something causes the email address to fall [...]
```
...it feels somewhere between the two)